bin/pgenv

```diff
@@ -1171,29 +1253,16 @@ EOF
 
     config|configuration)
         action=$2
-        v=$3
 
-        # which version does the user wants?
-        case "$v" in
-            version|current)
-                v=$( pgenv_current_postgresql_version )
-                title='PostgreSQL currently in use'
-                ;;
-            default)
-                v=''
-                title='Default configuration'
```

Comment:
So no more title for the default version?

Reply:
Seems everything is in place for filtering now.